transformations/longer_location_ner/transformation.py

```diff
@@ -0,0 +1,83 @@
+from typing import List
+from random import choice, seed, getrandbits
+from interfaces.TaggingOperation import TaggingOperation
+from tasks.TaskTypes import TaskType
+
+"""
+A tagging implementation of NER systems.
+I am travelling to London. --> I am travelling to South/Central London.
+"""
+
+
+def pick_random_word():
+    pre_word_list = ["East", "West", "South", "North", "Eastern", "Western", "Northern", "Central", "New"]
+    post_word_list = ["City", "Republic", "Beach"]
+    if bool(getrandbits(1)):
+        pertub_type = "pre"
+        selected_word = choice(pre_word_list)
+    else:
+        pertub_type = "post"
+        selected_word = choice(post_word_list)
+    return selected_word, pertub_type
+
+
+def create_token_and_tag(b_index, i_index, b_tag, i_tag, token_seq, tag_seq):
+    word, pertub_type = pick_random_word()
+    if pertub_type == "pre":
+        token_seq.insert(b_index, word)
+        tag_seq.insert(b_index, b_tag)
+        tag_seq[i_index] = i_tag
+    elif pertub_type == "post":
+        token_seq.insert(b_index+1, word)
+        tag_seq.insert(b_index+1, i_tag)
+    return token_seq, tag_seq
+
+
+class LongerLocationNer(TaggingOperation):
+    tasks = [TaskType.TEXT_TAGGING]
+    languages = "All"
+
+    def __init__(self):
```

Comment:
Please take max_output as one of the arguments, which decides the max number of outputs the transformation will return.

Reply:
done